app/main.py

```diff
@@ -1,8 +1,29 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    list_of_people = [
+        Person(person_data["name"], person_data["age"])
+        for person_data in people
+    ]
+
+    for person_data in people:
+        person = Person.people[person_data["name"]]
+
+        wife = Person.people.get(person_data.get("wife"))
+        if wife is not None:
+            person.wife = wife
+            wife.husband = person
```

Comment:
```suggestion
        if person_data.get("wife"):
            person.wife = Person.people[person_data["wife"]]
```
You can do like that for wife and husband

Reply:
`if person_data.get("wife"): `# check if there is a wife for person in person_data dict
`    person.wife = Person.people[person_data["wife"]]`  # assign to Person instance new wife attribute with wifes Person instance